src/app/components/jsx-helpers/raw-html.tsx

```diff
@@ -16,7 +16,9 @@ function activateScripts(el: HTMLElement) {
 
         Array.from(s.attributes)
             .forEach((a) => newScript.setAttribute(a.name, a.value));
-        newScript.appendChild(document.createTextNode(s.textContent));
+        if (s.textContent) {
```

Comment:
you should never hit this condition because of the other one you added above right?

Reply:
That is what I had to split up. I was testing for both `s` and `textContent` at the top, but the `textContent` test had to be moved down.